src/core/tests/pass/serialization/from_model.cpp

```diff
@@ -12,6 +12,11 @@
 #include "openvino/util/file_util.hpp"
 #include "read_ir.hpp"
 #include "util/test_common.hpp"
+#include "openvino/op/split.hpp"
```

Comment:
-> `#include "openvino/op/ops.hpp"`

Reply:
Hi 
I changed opsets to version and deleted opset headers to speed up a compilation. If I add #include "openvino/op/ops.hpp" to all cpp files, it will be equal to using opsets.